.github/workflows/pytest.yml

```diff
@@ -0,0 +1,29 @@
+name: Python Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.10']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install pytest
+        pip install -r requirements_dev.txt
+        pip install -r requirements_test.txt
+    - name: Test with pytest
+      run: |
+        pytest tests/
```

Comment:
```
  /opt/hostedtoolcache/Python/3.9.10/x64/lib/python3.9/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: asyncio_mode
```

could this be related to your previous PR? Should we remove that mode?

Reply:
Ah it looks like this option is only available for pytest-asyncio>=0.17 https://github.com/pytest-dev/pytest-asyncio#modes -- but maybe its not installed. Let me try adding as an explicit dependency.

Reply:
I went ahead and removed this. The original motivation was for client library tests, before adding the home assistant test harness, so you're right we should just remove this mode. Can add back later if needed.